lms/djangoapps/courseware/model_data.py

```diff
@@ -240,206 +629,133 @@ def _fields_to_cache(self, descriptors):
                 scope_map[field.scope].add(field)
         return scope_map
 
-    def _cache_key_from_kvs_key(self, key):
-        """
-        Return the key used in the FieldDataCache for the specified KeyValueStore key
-        """
-        if key.scope == Scope.user_state:
-            return (key.scope, key.block_scope_id)
-        elif key.scope == Scope.user_state_summary:
-            return (key.scope, key.block_scope_id, key.field_name)
-        elif key.scope == Scope.preferences:
-            return (key.scope, BlockTypeKeyV1(key.block_family, key.block_scope_id), key.field_name)
-        elif key.scope == Scope.user_info:
-            return (key.scope, key.field_name)
-
-    def _cache_key_from_field_object(self, scope, field_object):
-        """
-        Return the key used in the FieldDataCache for the specified scope and
-        field
-        """
-        if scope == Scope.user_state:
-            return (scope, field_object.module_state_key.map_into_course(self.course_id))
-        elif scope == Scope.user_state_summary:
-            return (scope, field_object.usage_id.map_into_course(self.course_id), field_object.field_name)
-        elif scope == Scope.preferences:
-            return (scope, field_object.module_type, field_object.field_name)
-        elif scope == Scope.user_info:
-            return (scope, field_object.field_name)
-
-    def find(self, key):
+    @contract(key=DjangoKeyValueStore.Key)
+    def get(self, key):
         '''
```

Comment:
I wish pylint insisted on triple double-quotes for docstrings - or at least enforced either that or triple single-quotes for all docstrings. I've been converting them to triple double-quotes everywhere. (They show up as different colors in my syntax highlighting...)